### DISCUSSION TITLE
not working for Windows 11 professional version code 1000.22000.438.0

### DISCUSSION THREAD

**Discussion by alleniver** [valinet/ExplorerPatcher#696] [General]
hi ,i have installed the last explorerpatcher,but not working,nothing to show???

**Comment by valinet** [comment]
Just tested this in a brand new Windows 11 VM with build 22000.438, seems to work just fine:

![image](https://user-images.githubusercontent.com/6503598/150231170-32d92bcb-2bb7-4671-9e11-cc1a01f58721.png)

**Reply by alleniver** [reply]
maybe my fault,thank you!
